### PR TITLE
updated to fix bug where req.query.watermark equaled the string "null"

### PR DESCRIPTION
**Pull request by User1m**
working with offline_dl this week we encountered an issue where `req.query.watermark` equaled the string `"null"`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "tsc",
     "watch": "tsc -w",
     "nodewatch": "nodemon index.js",
-    "start": "node dist/index.js",
+    "start": "node dist/cmdutil.js",
```
Comment (ryanvolum) [inline package.json]: @User1m not sure why you changed the start script to the cmdutil.js. That file was actually for running the connector from the command line, which I ended up deprecating. Did you mean to include this in your PR?

File: src/bridge.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export const initializeRoutes = (app: express.Server, serviceUrl: string, botUrl
 
     //Gets activities from store (local history array for now)
     app.get('/directline/conversations/:conversationId/activities', (req, res) => {
-        let watermark = Number(req.query.watermark || 0);
+        let watermark = req.query.watermark && req.query.watermark !== "null" ? Number(req.query.watermark) : 0;
```
Comment (ryanvolum) [inline src/bridge.ts]: @User1m I can get behind this change! The other changes in this file seem like different spacing (to comments, etc).

Comment (User1m) [reply src/bridge.ts]: @ryanvolum yes. Will do

File: package.json (hunk 1 above)
Comment (User1m) [reply package.json]: @ryanvolum ohh. I assumed index was a left over reference bc there's no compiled index.js file. But I can remove this from the PR.

**Comment by User1m** [comment]
@ryanvolum updates made

File: src/bridge.ts (hunk 2 above)
Comment (ryanvolum) [reply src/bridge.ts]: gotcha - you using vscode? I thought I had everything up to its (standard) standards...

**Pull request merged**